src/platform/plugins/shared/workflows_extensions/test/scout/api/tests/approved_step_definitions.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
```

Comment:
nit: is it possible to move this file to `src/platform/plugins/shared/workflows_extensions/test/scout/api/fixtures` ?

We are using test discovery to balance tests on CI and ideally only spec files should be in `/tests` directory